payments/vipps/redirect.md

```diff
@@ -58,32 +30,28 @@ consumers mobile number.
 [Vipps-screenshot-1]{:width="426px" :height="632px"}
 ![Vipps Payments][Vipps-screenshot-2]{:width="427px" :height="694px"}
 
-### API Requests
+## Operations
 
 The API requests are displayed in the [purchase flow](#purchase-flow).
-The options you can choose from when creating a payment with key `operation`
-set to Value `Purchase` are listed below.
-
-### Options before posting a payment
-
-All valid options when posting a payment with operation equal to Purchase,
-are described in [the technical reference][vipps-payments].
+You can create a Vipps payment with the `operation` value set to `purchase`. See
+the `purchase` example below.
 
-#### Type of authorization (Intent)
+### Intent
 
-**Authorization (two-phase)**: The intent of a Vipps purchase is always
-`Authorization`.
-The amount will be reserved but not charged.
+**`Authorization` (two-phase)**: The intent of the payment identifies how and
+when the charge will be effectuated. This determines the type of transaction
+used during the payment process. The intent of a Vipps purchase is always
+`Authorization`. The amount will be reserved but not charged.
 You will later (i.e. if a physical product, when you are ready to ship the
-purchased products) have to make a [Capture][captures] or
-[Cancel][cancellations] request.
+purchased products) have to make a [Capture][capture] or
+[Cancel][cancel] request.
 
-#### General
-
-**Defining CallbackURL**: When implementing a scenario, it is optional to set
-a [`CallbackURL`][callbackurl] in the `POST` request.
-If callbackURL is set Swedbank Pay will send a postback request to this URL
-when the consumer has fulfilled the payment.
+{% include alert.html type="success" icon="link" body="**Defining
+`callbackUrl`**: When implementing a scenario, it is optional to set a
+`callbackUrl` in the `POST` request. If `callbackUrl` is set, Swedbank Pay will
+send a `POST` request to this URL when the consumer has fulfilled the payment.
+[See the Callback API description for more
+information](/payments/vipps/other-features#callback)." %}
 
 ## Purchase flow
```

Comment:
When you change both PayEx_FrontEnd and PayEx_BackEnd to SwedbankPay, you are technically going from two participants to one participant. I suggest to change them to SwedbankPay_FrontEnd and SwedbankPay_BackEnd